roles/kubernetes/node/templates/kubelet.docker.service.j2

```diff
@@ -23,8 +23,6 @@ ExecStart={{ bin_dir }}/kubelet \
 		$DOCKER_SOCKET \
 		$KUBELET_NETWORK_PLUGIN \
 		$KUBELET_CLOUDPROVIDER
-ExecStartPre=-{{ docker_bin_dir }}/docker rm -f kubelet
```

Comment:
I think it'd be good to keep these in here and just put a conditional around it --   
`if  kubelet_deployment_type == docker `